Dockerfile

```diff
@@ -4,14 +4,14 @@ RUN apt-get update -qq && apt-get upgrade -y
 RUN curl -sL https://deb.nodesource.com/setup_9.x | bash -
 RUN apt-get install -y build-essential libpq-dev libxml2-dev libxslt1-dev \
     libfontconfig1 libfontconfig1-dev nodejs unzip xvfb chromium chromium-driver fonts-dejavu && \
-  apt-get clean
+    apt-get clean
 
-# Install Google Chrome
-RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
-RUN sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
-RUN apt-get update && apt-get install -y google-chrome-stable
+## Install Google Chrome
+#RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
+#RUN sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
+#RUN apt-get update && apt-get install -y google-chrome-stable
 
-RUN echo "73.0.3683.68" > /root/.chromedriver-version
+#RUN echo "73.0.3683.68" > /root/.chromedriver-version
```

Comment:
Should this be deleted rather than commented out?

Reply:
yep....should be done now.